WISHLIST.md

```diff
@@ -0,0 +1,64 @@
+WISHLIST to make LUYA a better CMS
+==============
+
+Simply put your suggestions and wishes for improvements in this basic text file to help us to make LUYA your favorite futur CMS.
+
+
+Installation
+-------------------
+
+- integrate composer fxp/composer-asset-plugin installation and availibilty test in composer create project.
+
+
+
+Console
+-------------------
+- beautify console appearance and shorten luya console commands (tab complete doesn't work OSX)
+- numeric select option for selections
+
+
+
+> Questions / Ideas
+
+Is it the best choice to use yii2 console? <br/>
```

Comment:
what do you mean? in luya/yii2 you can build console commands within seconds, just add 1 file. read more on luya.io